em_stress/em_stress.py.data/cpu_hotplug.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+# Copyright: 2016 IBM
+# Author: Pavithra D P <pavithra@linux.vnet.ibm.com>
+
+import os
+import subprocess
+
+
+class cpu_hotplug():
```

Comment:
its better to use `class set_cpu_online():` as hotplug might get misinterpreted.